acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -0,0 +1,930 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) MONAI Consortium  \n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\");  \n",
+    "you may not use this file except in compliance with the License.  \n",
+    "You may obtain a copy of the License at  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;http://www.apache.org/licenses/LICENSE-2.0  \n",
+    "Unless required by applicable law or agreed to in writing, software  \n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,  \n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  \n",
+    "See the License for the specific language governing permissions and  \n",
+    "limitations under the License.\n",
+    "\n",
+    "# Benchmark TensorRT models inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial shows how to convert a bundle in the MONAI model zoo to a TensorRT engine-based TorchScript for better performance and benchmark the difference between the PyTorch module and the TensorRT engine-based TorchScript.  \n",
```

Comment:
Please don't describe "model zoo" in the document, use "This tutorial shows how to convert the model of a release bundle to a TensorRT engine-based TorchScript for better performance and benchmark the difference between the PyTorch native model and the TensorRT engine-based TorchScript."

Reply:
convert the model of a release bundle >  convert the model of a **released** bundle